components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -1029,4 +1034,16 @@ private void setAuthenticatedIDPsOfApp(SessionContext sessionContext,
         }
         sessionContext.setAuthenticatedIdPsOfApp(applicationName, authenticatedIdPDataMap);
     }
+
+    private void addAuditLogs(String sessionKey, String tenantDomain, String traceId, Long createdTimestamp) {
+
+        JSONObject auditData = new JSONObject();
+        auditData.put(SessionMgtConstants.SESSION_CONTEXT_ID, sessionKey);
+        auditData.put(SessionMgtConstants.TENANT_DOMAIN, tenantDomain);
+        auditData.put(SessionMgtConstants.TRACE_ID, traceId);
+        auditData.put(SessionMgtConstants.SESSION_CREATED_TIMESTAMP, createdTimestamp);
```

Comment:
As we discussed offline, at this point we cannot add the expiry time because the expiry time is calculated from a following step. Hense added the remember me flag to the audit log.